packages/frames.js/package.json

```diff
@@ -58,14 +58,21 @@
       "require": "./dist/next/server.js",
       "import": "./dist/next/server.js",
       "types": "./dist/next/server.d.ts"
+    },
+    "./xmtp": {
+      "require": "./dist/xmtp/index.js",
+      "import": "./dist/xmtp/index.js",
+      "types": "./dist/xmtp/index.d.ts"
     }
   },
   "devDependencies": {
+    "@xmtp/frames-validator": "^0.5.0",
     "tsup": "^8.0.1",
     "typescript": "^5.3.3"
   },
   "license": "MIT",
   "peerDependencies": {
+    "@xmtp/frames-validator": "^0.5.0",
```

Comment:
will people get an error of an unmet peer dependency?

Reply:
can confirm this is true. if they don't try to import the package it won't give any problems.